packages/lib/deriveAppDictKeyFromType.ts

```diff
@@ -21,6 +21,12 @@ export function deriveAppDictKeyFromType(appType: string, dict: Record<string, u
     return appTypeVariant2;
   }
 
+  const appTypeVariant3 = appType.replace(/_/g, "");
+  handlers = dict[appTypeVariant3];
+  if (handlers) {
+    return appTypeVariant3;
+  }
+
```

Comment:
can you explain what this does? maybe with a comment above

Reply:
A comment on the code itself would be helpful @leog

Reply:
Done as well 👍